style.css

```diff
@@ -0,0 +1,436 @@
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    background: #fff;
+    color: #000;
+    font-size: 15px;
+    line-height: 1.5;
+  }
+
+.medical-header__container{
+   background: #FFF5F0;
+ 
+}
+.medical-header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 60px;
+    padding: 20px 0;
+    font-size: 13px;
+    width: 70%;
+    max-width: 1100px;
+    margin: auto;
+    
+  
+}
+
+.medical-header__logo .medical-header__anchor{
```

Comment:
this is not bem you try to repete your self here `.medical-header__logo .medical-header__anchor`.